setup.py

```diff
@@ -8,6 +8,8 @@
 import ah_bootstrap
 from setuptools import setup
 
+ah_bootstrap._check_submodule("asdf-standard")
+
```

Comment:
No, and it's not even a public function.  I plan to file a PR against astropy-helpers/package-template to make this public (since it's useful).